docs/user_guide/logging.rst

```diff
@@ -0,0 +1,63 @@
+Logging
+=======
+
+Since version 0.18.0 :program:`rebase-helper` uses standard module-level loggers.
+
+There are also 3 loggers with a special purpose:
+
+=============================== =====================================================================
+Logger name                     Purpose
+=============================== =====================================================================
+:samp:`rebasehelper.traceback`  traceback generated by unhandled exceptions
+:samp:`rebasehelper.summary`    messages forming the rebase summary
+:samp:`rebasehelper.report`     messages forming the textual rebase report (used by text output tool)
+=============================== =====================================================================
+
+
+:program:`rebase-helper` uses :class:`rebasehelper.logger.CustomLogger` logger class which provides extra
+logging levels.
+
+:mod:`rebasehelper.logger` module provides two utility functions to setup default handlers.
+:meth:`rebasehelper.logger.create_stream_handlers` sets up the default console handlers - with level INFO
+and a formatter displaying logging level for :samp:`rebasehelper` logger and with level DEBUG for
+:samp:`rebasehelper.summary` logger. Both of the handlers are instances of
+:class:`rebasehelper.logger.ColorizingStreameHandler` to provide colored output.
+:meth:`rebasehelper.logger.create_file_handlers` sets up the default file handlers for the main :samp:`rebasehelper`
+logger. 3 handlers are created, they differ in the logging level. The used logging levels are DEBUG, VERBOSE and INFO.
```

Comment:
This is too much poorly formatted information :wink:
Something like this should suffice:
```rst
:meth:`rebasehelper.logger.create_stream_handlers` sets up the default console handlers for :samp:`rebasehelper` and :samp:`rebasehelper.summary` loggers.
:meth:`rebasehelper.logger.create_file_handlers` sets up the default file handlers for :samp:`rebasehelper` logger, with INFO, VERBOSE and DEBUG levels. The respective log files are located in :file:`rebase-helper-results/logs/`.
```
It would probably be better to format it as a list or a table.